x/mm2/types/ticker.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/skip-mev/slinky/pkg/json"
+
+	slinkytypes "github.com/skip-mev/slinky/pkg/types"
+)
+
+const (
+	// DefaultMaxDecimals is the maximum number of decimals allowed for a ticker.
+	DefaultMaxDecimals = 36
+	// DefaultMinProviderCount is the minimum number of providers required for a
+	// ticker to be considered valid.
+	DefaultMinProviderCount = 1
+	// MaxMetadataJSONFieldLength is the maximum length of the MetadataJSON field.
```

Comment:
nit: what unit is this? In bytes or bits?

Reply:
bytes !